README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review Past Project
+
+> A project to microverse practice Code Review 2.0 training.
+
+## Built With
+
+- HTML
+- CSS
+- VS-Code
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Author
```

Comment:
- [ ] You should provide the basic commands to set up, install, and use your project in these sub-sections. This is to guide your users on which command to run to avoid unnecessary stress. Also, it is not a good practice to assume that your users know what to do or which command to run. Lastly, remove sub-sections that are not related to the project (e.g. run tests).